app/lib/common/constants.dart

```diff
@@ -7,7 +7,10 @@ final annotationServerIp = Uri.https(
 );
 final annotationServerUrl = annotationServerIp.replace(path: 'api/v1');
 final labServerUrl = labServerIp.replace(path: 'api/v1');
-final keycloakUrl = labServerIp;
+final keycloakUrl = Uri.https(
+  'keycloak.k8s.kunst.me',
+  '',
+);
 final cpicMaxCacheTime = Duration(days: 90);
```

Comment:
Not sure we should put this in the repo but now it already is i guess

Did you ask Samuel about this?

Reply:
I guess it should be fine o.0.